cv/index.html

```diff
@@ -127,8 +127,8 @@ <h2>freeCodeCamp</h2>
                     <p class="subDetails">January 2019 - present</p>
                     <p><ul>
                           <li>Responsive Web Design Certificate.
-                              <ul><li>Demonstrates real-world page design from scratch: 
-                          <a href="https://codepen.io/collection/nRVGmW/">Projects</a></li></ul> 
+                              <ul class="sub-list"><li>Real-world page design from scratch: 

```

Comment:
What is the purpose of class="sub-list" here? Does it do something? No need to change it, just wondering.